system/HTTP/Negotiate.php

```diff
@@ -317,7 +317,7 @@ public function parseHeader(string $header): array
 	 *
 	 * @return boolean
 	 */
-	protected function match(array $acceptable, string $supported, bool $enforceTypes = false, $matchLocales = false): bool
+	public function match(array $acceptable, string $supported, bool $enforceTypes = false, $matchLocales = false): bool
```

Comment:
You cannot change the scope because it will break any classes that extend this: https://3v4l.org/13KBX

Need to find another way around.

Reply:
Noted, thanks!